runtime/runtime/src/congestion_control.rs

```diff
@@ -51,6 +51,14 @@ enum ReceiptForwarding {
     NotForwarded(Receipt),
 }
 
+pub(crate) struct DelayedReceiptQueueWrapper {
```

Comment:
Can you add a comment please? The fields are quite self explanatory but if there is any subtlety please add comments too.

Reply:
done